samples/snippets/src/main/java/com/example/bigquerystorage/WriteCommittedStream.java

```diff
@@ -45,54 +57,104 @@ public static void runWriteCommittedStream()
 
   public static void writeCommittedStream(String projectId, String datasetName, String tableName)
       throws DescriptorValidationException, InterruptedException, IOException {
+    // One time initialization.
```

Comment:
Maybe put the initalization code into a separate writer.initialize() method? I really like the structure of https://github.com/googleapis/java-bigquerystorage/pull/1585 from a docs perspective.

Reply:
Changed it back. Not sure how to do it in a way that both exemplifies the structure and is "Good Java"